app/hub/Views/SideNavigationView/SideNavigationViewContainer.jsx

```diff
@@ -66,7 +65,12 @@ class SideNavigationViewContainer extends Component {
 			{ id: 'logout', text: t('hub_side_navigation_log_out'), clickHandler: this._handleLogoutClick },
 		] : [
 			{ id: 'create-account', href: '/create-account', text: t('hub_side_navigation_create_account') },
-			{ id: 'log-id', href: '/log-in', text: t('hub_side_navigation_log_in') },
+			{
```

Comment:
make this line consistent with the other items in the array (one line vs multiple)

Reply:
there's too many key/value pairs for a one liner according to our linter